archetype/archetype-catalog.xml

```diff
@@ -66,10 +66,16 @@
 			<groupId>io.americanexpress.synapse</groupId>
 			<artifactId>synapse-archetype-client-rest-reactive-post</artifactId>
 			<version>0.2.19-SNAPSHOT</version>
-		</archetype><archetype>
+		</archetype>
+		<archetype>
 			<groupId>io.americanexpress.synapse</groupId>
 			<artifactId>synapse-archetype-client-rest-reactive-put</artifactId>
 			<version>0.2.19-SNAPSHOT</version>
 		</archetype>
+		<archetype>
+			<groupId>io.americanexpress.synapse</groupId>
+			<artifactId>synapse-archetype-service-rest-get</artifactId>
```

Comment:
should the module be named synapse-archetype-service-rest-reactive-get?

Reply:
there's a separate archetype that will be there for the reactive-get. So this one is correctly labeled